audio_rules.mk

```diff
@@ -1,6 +1,6 @@
 # This file contains rules for making assemblies for most music in the game.
 
-MID_ASM_DIR := $(MID_SUBDIR)
+MID_ASM_DIR := $(MID_BUILDDIR)
```

Comment:
This is pretty opinionated. I thought it was better looking if we don't clutter the midi directory with the .s but I can change it back.